packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -560,6 +562,11 @@ export class LedgerKeyring extends EventEmitter {
     this.hdk = new HDKey();
   }
 
+  #arrayBufferToString(buffer: ArrayBuffer): string {
```

Comment:
This is created to work in both web and mobile native environment, some common way to use TextDecoder() only work for web and not compatible with mobile native.